### DISCUSSION TITLE
Where is Input Image?

### DISCUSSION THREAD

**Discussion by gerroon** [AIGODLIKE/ComfyUI-BlenderAI-node#103] [Q&A]
Hi

The doc says there isa  node called Input Image but I cant find it in the menus oor the search

**Comment by cnk4cnk** [comment]
I also don't have these nodes. I have macOS. I installed ComfyUI a long time ago and it works great. I installed this plugin for Blender versions 3.6, 4.0, 4.1. Specified the path to an existing Python directory.

The renderer works in all versions, I can start the server and work. 
But there are no special nodes of the plugin itself. For example "Input Image" or "Save". 
Even in the node creation menu, there is no corresponding menu section.

I tried creating a separate "venv" as recommended for Linux and install new Python. And specify the path there in the plugin. The server does not start. But in the node creation menu, the “Blender” section appears, in which there are plugin nodes. But since the server is not running, all the nodes are not working. 

What can I do? How to fix it?

**Reply by zengels** [reply]
the node name is in Chinese that's why "input image" is not reachable in Blender with Shift+A . search in ADD and "blender"  nodes in Blender ComfyUI Nodes window .  In Fact i Just find it but i don't know what node is the Input image ... 😂. as it is in chinese
